po/de.po

```diff
@@ -782,23 +972,42 @@ msgstr "Nächster Monat"
 msgid "next year"
 msgstr "Nächstes Jahr"
 
-#: GTG/core/dates.py:384
+#: GTG/core/dates.py:384 GTG/core/dates.py:512
 msgid "Monday"
 msgstr "Montag"
 
-#: GTG/core/dates.py:390
+#: GTG/core/dates.py:390 GTG/core/dates.py:518
 msgid "Sunday"
 msgstr "Sonntag"
 
-#: GTG/core/dates.py:452
+#: GTG/core/dates.py:499
+msgid "day"
+msgstr "tag"
+
+#: GTG/core/dates.py:501
+msgid "other-day"
+msgstr "zweiten-tag"
+
+#: GTG/core/dates.py:503
+msgid "week"
+msgstr "woche"
+
+#: GTG/core/dates.py:505
+msgid "month"
+msgstr "Monat"
+
+#: GTG/core/dates.py:507
+msgid "year"
+msgstr "jahr"
```

Comment:
I didn't check where these strings are being used, but I think they should be uppercase, like "Tag" instead of "tag". Nouns are always uppercase in German.

Reply:
They are made lowercase anyway later, but I could indeed make them uppercase (since it shouldn't anything functionally wise)
https://github.com/getting-things-gnome/gtg/blob/e330d27d966962aa4f41b1532cf8f6869babbbe2/GTG/core/dates.py#L495-L508

Reply:
Nevermind then. I guess you can leave it like that.

Reply:
Made them uppercase anyway